Resources/Prototypes/Objectives/traitor.yml

```
@@ -333,3 +333,25 @@
+  components:
+  - type: Objective
+    difficulty: 2.5
+  - type: NotJobRequirement
+    job: NanotrasenRepresentative
+
+- type: entity
+  categories: [HideSpawnMenu]
+  parent: BaseNTRepObjective
+  id: NTRepGunStealObjective
+  components:
+  - type: StealCondition
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)